package summary.dohyun.chapter08.Excercise;

public class Exercise4 {
	// 문제 보고 헷갈려서 정리 한것

	// throws 예약어를 시그내처에 추가하면 그 메소드를 호출하는 곳에서 예외처리해야댐 호출한쪽으로 떠넘기는개념
	// throw : 강제로 예외를 발생시키는 경우에 사용한다.

	// 답 = c, Exception은 모든예외처리의 조상이라고 알고있는데 첫번째 catch블럭에서 Exception을
	// 처리하면 다음 블록에서 Exception처리를 제외한 예외처리가되야 하므로 오류!

}
